docs/sql-ref-syntax-ddl-alter-table.md

```diff
@@ -23,7 +23,7 @@ license: |
 `ALTER TABLE` statement changes the schema or properties of a table.
 
 ### RENAME 
-`ALTER TABLE RENAME` statement changes the table name of an existing table in the database.
+`ALTER TABLE RENAME TO` statement changes the table name of an existing table in the database.
```

Comment:
Are these changes meant to be in this PR? looks somewhat unrelated

Reply:
Sorry, I didn't realize this went in. Will remove now.